tests/integration/test_raw_persistence.py

```
@@ -0,0 +1,305 @@
+        if response.status_code == 200:
+            response_data = response.json()
+            ingestion_id = response_data["ingestion_id"]
+            
+            raw_record = test_db_session.query(RawIngestion).filter_by(
+                ingestion_id=ingestion_id
+            ).first()
+            
+            assert raw_record is not None
+            assert raw_record.source_type == "python-repo"
+            assert raw_record.content_type == "repository"
+            assert raw_record.metadata["source_path"] == "/tmp/test-repo"
```

Ruff (0.14.11)

Probable insecure usage of temporary file or directory: "/tmp/test-repo"

(S108)